src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -713,6 +714,8 @@ export class CellModel extends Disposable implements ICellModel {
 			// Serialize cell output once the cell is done executing
 			this.sendChangeToNotebook(NotebookChangeType.CellOutputUpdated);
 			this.notifyExecutionComplete();
+			// Allow screen reader to announce when a cell is done running
+			alert(localize('cellExecutionComplete', "Cell execution is complete"));
```

Comment:
Do we also alert when cell execution started? Similar to loading (although maybe less necessary since cell execution is kicked off by the user...but I still think it'd be a nice thing to have just to be explicit)

Reply:
Currently we don't alert when cell execution is started - I can add that.